Makes the hexen2 game to directly activate its mission pack support,
if it is installed. The -portals or -h2mp command line argument will
not be necessary. You will need to use the -noportals command line
argument to disable mission pack support.

diff -urNp hexen2source-1.4.1/hexen2/quakedef.h hexen2source-1.4.1x/hexen2/quakedef.h
--- hexen2source-1.4.1/hexen2/quakedef.h	2006-05-19 16:19:47.000000000 +0300
+++ hexen2source-1.4.1x/hexen2/quakedef.h	2006-05-19 16:25:24.000000000 +0300
@@ -50,7 +50,7 @@
 // to activate it.  If you want direct activation, change the below
 // undef to a define : in that case, player must use the -noportals
 // command line argument to disable mission pack support.
-#undef	H2MP
+#define	H2MP
 
 #include <sys/types.h>
 #include <math.h>
